addons/mail/static/src/web/thread_service_patch.js

```diff
@@ -149,8 +149,11 @@ patch(ThreadService.prototype, "mail/web", {
     async insertSuggestedRecipients(thread, dataList) {
         const recipients = [];
         for (const data of dataList) {
-            const [partner_id, emailInfo, lang, reason] = data;
-            const [name, email] = emailInfo && parseEmail(emailInfo);
+            const [partner_id, emailInfo, lang, reason, defaultCreateValues] = data;
+            let [name, email] = emailInfo && parseEmail(emailInfo);
```

Comment:
(not exactly related to your PR)
I think this line tries to be too smart and ends up being a bit shaky. If `emailInfo` is falsy and is anything other than an iterable (which includes `null`, `undefined`, `false`), trying to destructure it will result in an error. I believe it would be more robust code to either use a regular `if` statement, or, if we're going to keep it a one-liner, perhaps use a ternary operator and return an empty array when `emailInfo` is falsy